<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>udpClientB</class>
 <widget class="QWidget" name="udpClientB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>连接B</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="IPlabel">
     <property name="text">
      <string>对方IP地址：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="IPlineEdit">
     <property name="text">
      <string>255.255.255.255</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="joinpushButton">
     <property name="text">
      <string>加入组播</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="PORTlabel">
     <property name="text">
      <string>对方端口号：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="PORTlineEdit">
     <property name="text">
      <string>8888</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="leavepushButton">
     <property name="text">
      <string>离开组播</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QTextEdit" name="readtextEdit"/>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QTextEdit" name="writetextEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="sendpushButton">
     <property name="text">
      <string>send</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="closepushButton">
     <property name="text">
      <string>close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
